packages/components/inputs/select-utils/src/create-select-styles.ts

```diff
@@ -211,7 +211,7 @@ const controlStyles = (props: TProps) => (base: TBase, state: TState) => {
       backgroundColor: (() => {
         if (!props.isDisabled && !props.isReadOnly) {
           if (props.appearance === 'quiet') {
-            return designTokens.backgroundColorForInputAsQuietWhenHovered;
+            return 'hsla(240, 64%, 58%, 4%)';
```

Comment:
This is the same as the `color-primary` design token plus `alpha` 4%.
I'm wondering if there's any way we can leverage the existing design token.

Reply:
Yes this one is a bit tricky. I was thinking maybe I could pass the alpha value and form a hsla like this `hsla(${designTokens.colorPrimary}, 4%)` but this will output `(hsl(240, 64%, 58%), 4%)`.
Perhaps I could write a function to extract the value. What do you think?

Reply:
I tried the color-mix but it seems like we will likely be circling back to the complexity we are trying to avoid.
`background-color: color-mix(in hsl, var(--color-primary), hsla(0deg, 0%, 0%, 4%));`  but it gives a different result from what we want to achieve.
<img width="115" alt="image" src="https://github.com/commercetools/ui-kit/assets/19975842/854b2f62-cebe-4d79-823b-343382df1758">

Reply:
Yes, you're right it doesn't seem like an feasible solution.
We can however add a comment to let our future selves about this is the primary color with an `alpha` channel we only need here.

Thanks for giving it a try.

Reply:
updated here: https://github.com/commercetools/ui-kit/pull/2785/commits/065e27db2ab90e9c6103bb78679b8c7d9ee5283f
Thank you